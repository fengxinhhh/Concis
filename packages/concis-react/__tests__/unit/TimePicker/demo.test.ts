import demoTest from '../../demoTest';
demoTest('TimePicker');
