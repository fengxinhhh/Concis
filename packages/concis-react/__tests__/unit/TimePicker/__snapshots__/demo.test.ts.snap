// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimePicker demo render TimePicker/demos//Users/apple/Desktop/Concis/packages/concis-react/src/TimePicker/demos/index2.tsx correctly 1`] = `
LoadedCheerio {
  "0": Document {
    "children": Array [],
    "endIndex": null,
    "next": null,
    "parent": null,
    "prev": null,
    "startIndex": null,
    "type": "root",
  },
  "_root": LoadedCheerio {
    "0": Document {
      "children": Array [],
      "endIndex": null,
      "next": null,
      "parent": null,
      "prev": null,
      "startIndex": null,
      "type": "root",
    },
    "_root": [Circular],
    "length": 1,
    "options": Object {
      "decodeEntities": true,
      "xml": false,
    },
  },
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "xml": false,
  },
}
`;

exports[`TimePicker demo render TimePicker/demos//Users/apple/Desktop/Concis/packages/concis-react/src/TimePicker/demos/index3.tsx correctly 1`] = `
LoadedCheerio {
  "0": Document {
    "children": Array [],
    "endIndex": null,
    "next": null,
    "parent": null,
    "prev": null,
    "startIndex": null,
    "type": "root",
  },
  "_root": LoadedCheerio {
    "0": Document {
      "children": Array [],
      "endIndex": null,
      "next": null,
      "parent": null,
      "prev": null,
      "startIndex": null,
      "type": "root",
    },
    "_root": [Circular],
    "length": 1,
    "options": Object {
      "decodeEntities": true,
      "xml": false,
    },
  },
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "xml": false,
  },
}
`;

exports[`TimePicker demo render TimePicker/demos//Users/apple/Desktop/Concis/packages/concis-react/src/TimePicker/demos/index4.tsx correctly 1`] = `
LoadedCheerio {
  "0": Document {
    "children": Array [],
    "endIndex": null,
    "next": null,
    "parent": null,
    "prev": null,
    "startIndex": null,
    "type": "root",
  },
  "_root": LoadedCheerio {
    "0": Document {
      "children": Array [],
      "endIndex": null,
      "next": null,
      "parent": null,
      "prev": null,
      "startIndex": null,
      "type": "root",
    },
    "_root": [Circular],
    "length": 1,
    "options": Object {
      "decodeEntities": true,
      "xml": false,
    },
  },
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "xml": false,
  },
}
`;
